docs/cache/pattern/api-caching.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Caching API calls to WolframAlpha using Momento Cache
+sidebar_label: Caching WolframAlpa API calls
+description: Learn how to use Momento Cache to boost performance of applications using the WolframAlpha API
+keywords:
+ - cache
+ - momento
+ - read-aside
+ - wolframalpha
+ - api caching
+ - api
+---
+
+# Use Momento Cache to cache results from an API
+
+Remote calls to an API can be slow and expensive. Depending on the access pattern, it can be desirable to store results in a cache to improve your application's latency and availability.
+
+With [Momento Cache](../), you can easily cache results from an API in a highly-available and scalable remote cache.
+
+## Getting Started
+
+In this example we will build an application that can give us a string description of the current weather of the given place. We will use WolframAlpha API in this example to show you how we can cache the results of an API.
+
+Weather data is relatively static over short periods, making it an ideal candidate for caching. Additionally WolframAlpha API can be slow to respond due to its computational model. You can expect latency of around 700ms for WolframAlpha API versus the latency of less than 5ms for Momento GET API.
```

Comment:
It feels like you're missing the word "the" in a couple spots:
* "the WolframAlpha API can be slow..."
* "less than 5ms for the Momento GET API"

Reply:
Same on line 28, "to call the WolframAlpha API"